src/components/LHNOptionsList/OptionRowLHNData.tsx

```diff
@@ -24,7 +24,7 @@ function OptionRowLHNData({
     receiptTransactions,
     parentReportAction,
     transaction,
-    lastReportActionTransaction = {},
+    lastReportActionTransaction = null,
```

Comment:
We don't need null/undefined default values (please do the same for other places in your code if applicable):
```suggestion
    lastReportActionTransaction,
```